pkg/pool-weighted/contracts/lib/CircuitBreakerLib.sol

```diff
@@ -109,7 +109,7 @@ library CircuitBreakerLib {
     uint256 private constant _BOUND_SHIFT_BITS = 64 - _BOUND_WIDTH;
 
     /**
-     * @notice Returns the BPT price and weight complement values, and lower and upper bounds for a given token.
+     * @notice Returns the BPT price and normalized weight values, and lower and upper bounds for a given token.
```

Comment:
This makes it clear that we're missing some term here, since this is most definitely _not_ the token's weight.

Reply:
It is though - I'm directly storing and retrieving the normalized weight now, and only taking the complement inside when I compute the ratio.

Reply:
I think Nico's point is that the value we're returning here doesn't have to be the current weight of the token. It's the weight of the token at the point at which we cached the bound ratios, i.e. rather than `normalizedWeight` it should be `<something>Weight`

Reply:
`cacheWeight` maybe? Something to denote that it's "weighty" but is to test for whether we hit or miss the cache.

Reply:
Ok; was focused on complement vs weight, not stored weight vs "current" weight. I can standardize on "reference".